sentinel.go

```diff
@@ -114,7 +114,7 @@ func (opt *FailoverOptions) sentinelOptions(addr string) *Options {
 		Dialer:    opt.Dialer,
 		OnConnect: opt.OnConnect,
 
-		DB:       0,
+		DB:       opt.DB,
```

Comment:
These options are for Sentinel client. Do we really want to select a DB in Sentinel Server?